index.html

```diff
@@ -4012,6 +4023,16 @@ <h4>canonicalId</h4>
                   </p>
                 </div>
 
+                <div class="issue" title="Observance of canonicalId recommendations">
+                  <p>
+If a resolving party does not use the <code><a>canonicalId</a></code> value
+as its primary ID value for the DID subject and treat all other equivalent
+values as secondary aliases, there may be negative or unexpected issues that 
+arise related to user experience. You are strongly advised to observe the 
```

Comment:
```suggestion
arise related to user experience. Implementers are strongly advised to observe the 
```

Make the 'You' party more explicit.